webapp/src/components/sidebar/ModelLibrary.vue

```diff
@@ -157,6 +157,8 @@ memoryStore.$subscribe((_, state) => {
 
 const memory = memoryStore.useIndexedDB ? new IndexedDB() : new EmptyMemory();
 
+let tempDeletedModel : { id : string; model : Model, metadata : ModelMetadata} | undefined;
```

Comment:
@tharvik do you recommend using a ref here?

Reply:
hum, that shouldn't matter as it is not derived upon (no `compute`, no ui changes.

but it can be avoided if used as an argument to `undoDeleteModel`

Reply:
it was resolved by another branch this week